uninstall.sh

```diff
@@ -73,6 +73,29 @@ if [ "$doUninstall" == true ]; then
 		echo -e "No"
 	fi
 
+# Decky Loader
+
+if [[ -f "$HOME/homebrew/services/PluginLoader" ]] ; then
+
+	text="`printf "An installation of Decky Loader was found on your system. Would you like to open the Decky Loader Uninstallation Tool?"`"
+
+	zenity --question \
+		 --title="Decky Loader" \
+		 --width=450 \
+		 --ok-label="Launch the Decky Loader Uninstallation Tool" \
+		 --cancel-label="No, continue with uninstalling EmuDeck" \
+		 --text="${text}"
+	ans=$?
+	if [ $ans -eq 0 ]; then
+		curl -S -s -L -O --output-dir $EMUDECKGIT --connect-timeout 30 https://github.com/SteamDeckHomebrew/decky-installer/releases/latest/download/user_install_script.sh
```

Comment:
I don't think you want to output this to the emudeck git folder, as git clean will remove it. Perhaps it makes more sense to put it in the ~/emudeck folder? Maybe i am overthinking it.